index.js

```diff
@@ -337,7 +385,12 @@ module.exports = function (schema, options) {
                 deletedId: true,
             }
         };
+        
+        const options = {};
+        if (params.session) {
+            options.session = params.session;
+        }
 
-        return updateDocumentsByQuery(this, conditions, doc, callback);
+        return updateDocumentsByQuery(this, conditions, doc, options, callback);
```

Comment:
🤏 I'd go with `saveOptions` as a name here as well, that way if you ever need anything from options (setup when Mongoose Delete is added as a plugin) it'll be available.
```
        var saveOptions;
        if (params.session) {
            saveOptions = { session: params.session };
        }

        return updateDocumentsByQuery(this, conditions, doc, saveOptions, callback);
```